actions/add_station.lua

```diff
@@ -38,7 +38,7 @@ return function (node_info, fields, player)
 
 	if travelnet.is_falsey_string(owner_name) or owner_name == player_name or is_elevator then -- elevator networks
 		owner_name = player_name
-	elseif	not minetest.check_player_privs(player_name, { travelnet_attach=true })
```

Comment:
i _think_ this should work here too :thinking: 
```lua
minetest.check_player_privs(player_name, { [travelnet.attach_priv]=true })
```

Reply:
Yeah it should work but personally not sure it is much better compared to get_player_privs which in turn is a lot simpler under the hood.